frontend/src/container/ListOfDashboard/ImportJSON/index.tsx

```diff
@@ -82,6 +82,11 @@ function ImportJSON({
 
 			const dashboardData = JSON.parse(editorValue) as DashboardData;
 
+			// Add validation for uuid
+			if (dashboardData.uuid !== undefined && dashboardData.uuid.trim() === '') {
+				throw new Error('uuid cannot be an empty string');
```

Comment:
rather than blocking them here can we just add a new generated UUID and let them create the same ?

Reply:
@vikrantgupta25 - 

we just block them here for 2 cases:
- `uuid = ""` - empty uuid
- duplicate uuid
In all other case i.e. not adding any uuid i.e. uuid = undefined, we do allow to create the dashboard, and post that we add uuid from our side.
So either you don't add anything or add any unique uuid